package.json

```diff
@@ -9,23 +9,38 @@
     "heroku-postbuild": "npm run build",
     "precommit": "lint-staged",
     "start": "node lib/server.js",
-    "test": "tslint -p tsconfig.json"
+    "test": "tslint -p tsconfig.json --fix",
+    "webpack": "webpack --watch"
```

Comment:
If this is run as part of the dev server, you should add it to `Procfile.dev` instead of here